.dotnet.azure/sdk/openai/Azure.AI.OpenAI/tests/Azure.AI.OpenAI.Tests.csproj

```diff
@@ -15,6 +15,7 @@
     <PackageReference Include="NUnit3TestAdapter" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" />
     <PackageReference Include="Moq" />
+    <PackageReference Include="Newtonsoft.Json" />
```

Comment:
What introduced this as an explicit dependency?

Reply:
I added a comment to clarify why and how we can remove it in the future.